src/core/base/render_base.cpp

```diff
@@ -256,7 +255,7 @@ StdPicture AbstractRender_t::LoadPicture(const std::string &path,
     target.frame_h = static_cast<int>(h);
 
     uint8_t *textura = reinterpret_cast<uint8_t *>(FreeImage_GetBits(sourceImage));
-    XRender::loadTexture(target, w, h, textura, pitch);
+    g_render->loadTexture(target, w, h, textura, pitch);
```

Comment:
@ds-sloth, why did you replaced `XRender::` with direct `g_render->` calls here?